renderer/views/player.js

```diff
@@ -63,11 +64,39 @@ function renderPlayerControls (state, dispatch) {
         ${renderLoadingBar(state)}
         <div class="playback-cursor" style=${playbackCursorStyle}></div>
       </div>
-      <i class="icon play-pause" onclick=${() => dispatch('playPause')}>
-        ${state.video.isPaused ? 'play_arrow' : 'pause'}
+    `
+  ]
+  if (state.view.devices.chromecast) {
+    elements.push(hx`
+      <i.icon.chromecast
+        class="${!torrent.ready ? 'disabled' : ''}"
```

Comment:
This line shouldn't be necessary anymore, since the user can't get to the player before the torrent is ready anymore.

This was only necessary when the chromecast/airplay buttons were visible in the torrent list.